### ISSUE TITLE
Subdomain takeover through Propposify.com

### ISSUE THREAD

**Issue by barhaam** [EdOverflow/can-i-take-over-xyz#374]
##proposify.com

**Comment by knowthetech** [comment]
is it error page?

![image](https://github.com/EdOverflow/can-i-take-over-xyz/assets/52136037/771800b9-7439-4492-86e2-181fd932a480)

**Comment by barhaam** [comment]
@knowthetech yes but to show your content you actually have to make a proposal with POC, that will generate a token which will be visible on the takeovered subdomain.

**Comment by knowthetech** [comment]
> @knowthetech yes but to show your content you actually have to make a proposal with POC, that will generate a token which will be visible on the takeovered subdomain.

That has been checked already thanks for the reply.
